userpanel/modules/info/style/bclean/templates/updateuserinfobox.html

```diff
@@ -1,4 +1,7 @@
 {box title="Information about the subscriber"}
+
+XXXXXX
```

Comment:
A to co? Link do pornhub.com się zepsuł?

Reply:
Czy ten plik jest potrzebny? Nie mogę namierzyć jak go wywołać.

Reply:
```bash
# grep -r updateuserinfobox\.html userpanel
userpanel/modules/info/templates/updateuser.html:			{include file="module:updateuserinfobox.html"}
userpanel/modules/info/templates/updateuser.html:			{include file="module:updateuserinfobox.html"}```

Reply:
```bash
# grep -r updateuser\.html userpanel
userpanel/modules/info/functions.php:    $SMARTY->display('module:updateuser.html');
userpanel/modules/info/functions.php:		$SMARTY->display('module:updateuser.html');
```

Reply:
ok usunięty śmietek